config/environments/production.rb

```diff
@@ -61,6 +64,9 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
 
+  # Skip http-to-https redirect for the default health check endpoint.
+  # config.ssl_options = { redirect: { exclude: ->(request) { request.path == "/up" } } }
```

Comment:
We might want/need to uncomment this if we ever set up a healthcheck for the `web` Docker Compose service. For now, we'll leave it commented out.